scheduler-interface-spec.md

```diff
@@ -448,6 +448,21 @@ message AllocationAsk {
   bool placeholder = 10;
   // Is this ask the originator of the application?
   bool Originator = 11;
+  // The preemption policy for this ask
+  PreemptionPolicy preemptionPolicy = 12;
+}
+```
+
+Preemption policy:
+
+```protobuf
+message PreemptionPolicy {
+  // Allow preemption while running
```

Comment:
Can we rename these, and remove Kubernetes-specific terminology?

Maybe 'allowPreemptSelf', 'allowPreemptOther' (both boolean)